src/openrct2/ride/TrackData.cpp

```diff
@@ -285,6 +285,14 @@ static constexpr rct_track_coordinates TrackCoordinates[TrackElemType::Count] =
         {    0,    2,    0,    0,    0,   32 }, // TrackElemType::FlatTrack1x1B
         {    0,    2,    0,    0,    0,   32 }, // TrackElemType::FlatTrack1x4C
         {    0,    0,    0,   96,   32,    0 }, // TrackElemType::FlatTrack3x3
+        { 0, 2, 0, 248, -64, -32 }, // TrackElemType::LeftFlyerLargeHalfLoopUninvertedUp
+        { 0, 2, 0, 248, -64, 32 },  // TrackElemType::RightFlyerLargeHalfLoopUninvertedUp
+        { 0, 2, 0, -248, 64, -32 }, // TrackElemType::RightFlyerLargeHalfLoopInvertedDown
+        { 0, 2, 0, -248, 64, 32 },  // TrackElemType::LeftFlyerLargeHalfLoopInvertedDown
+        { 0, 2, 0, 280, -64, -32 }, // TrackElemType::LeftFlyerLargeHalfLoopInvertedUp
+        { 0, 2, 0, 280, -64, 32 },  // TrackElemType::RightFlyerLargeHalfLoopInvertedUp
+        { 0, 2, 0, -280, 64, -32 }, // TrackElemType::RightFlyerLargeHalfLoopUninvertedDown
+        { 0, 2, 0, -280, 64, 32 },  // TrackElemType::LeftFlyerLargeHalfLoopUninvertedDown
```

Comment:
This file still needs  beautifying to align with the style of the rest of the file.

Reply:
What exactly do you want me to do? By all accounts this is the style of the rest of the file.

```cpp
 { 0, 0, 0, 0, -32, 0 },     // ELEM_AIR_THRUST_TOP_CAP
        { 0, 0, 80, 0, 32, 0 },     // ELEM_AIR_THRUST_VERTICAL_DOWN
        { 0, 0, 240, 0, -160, 0 },  // ELEM_AIR_THRUST_VERTICAL_DOWN_TO_LEVEL
        { 0, 0, 0, 0, 0, 0 },       // ELEM_BLOCK_BRAKES
        { 0, 3, 0, 32, -32, -32 },  // ELEM_BANKED_LEFT_QUARTER_TURN_3_TILES_25_DEG_UP
        { 0, 1, 0, 32, -32, 32 },   // ELEM_BANKED_RIGHT_QUARTER_TURN_3_TILES_25_DEG_UP
        { 0, 3, 32, 0, -32, -32 },
        { 0, 1, 32, 0, -32, 32 },
        { 0, 3, 0, 64, -64, -64 },
        { 0, 1, 0, 64, -64, 64 },
        { 0, 3, 64, 0, -64, -64 },
        { 0, 1, 64, 0, -64, 64 },
        { 0, 0, 0, 16, 0, 0 },
        { 0, 0, 0, 16, 0, 0 },
        { 0, 0, 0, 16, 0, 0 },
        { 0, 0, 0, 16, 0, 0 },
        // ... ///
        {    0,    2,    0,    0,    0,   32 }, // TrackElemType::FlatTrack1x4A
        {    0,    2,    0,    0,    0,   32 }, // TrackElemType::FlatTrack2x2
        {    0,    2,    0,    0,    0,   32 }, // TrackElemType::FlatTrack4x4
        {    0,    2,    0,    0,    0,   32 }, // TrackElemType::FlatTrack2x4
        {    0,    2,    0,    0,    0,   32 }, // TrackElemType::FlatTrack1x5
        {    0,    2,    0,    0,    0,   32 }, // TrackElemType::FlatTrack1x1A
        {    0,    2,    0,    0,    0,   32 }, // TrackElemType::FlatTrack1x4B
        {    0,    2,    0,    0,    0,   32 }, // TrackElemType::FlatTrack1x1B
        {    0,    2,    0,    0,    0,   32 }, // TrackElemType::FlatTrack1x4C
        {    0,    0,    0,   96,   32,    0 }, // TrackElemType::FlatTrack3x3
        { 0, 2, 0, 248, -64, -32 }, // TrackElemType::LeftFlyerLargeHalfLoopUninvertedUp
        { 0, 2, 0, 248, -64, 32 },  // TrackElemType::RightFlyerLargeHalfLoopUninvertedUp
        { 0, 2, 0, -248, 64, -32 }, // TrackElemType::RightFlyerLargeHalfLoopInvertedDown
        { 0, 2, 0, -248, 64, 32 },  // TrackElemType::LeftFlyerLargeHalfLoopInvertedDown
        { 0, 2, 0, 280, -64, -32 }, // TrackElemType::LeftFlyerLargeHalfLoopInvertedUp
        { 0, 2, 0, 280, -64, 32 },  // TrackElemType::RightFlyerLargeHalfLoopInvertedUp
        { 0, 2, 0, -280, 64, -32 }, // TrackElemType::RightFlyerLargeHalfLoopUninvertedDown
        { 0, 2, 0, -280, 64, 32 },  // TrackElemType::LeftFlyerLargeHalfLoopUninvertedDown
```

Reply:
Okay i didn't see the first bit no need to change this little bit  It was a comment about the whole file though. TrackSequenceProperties the comments are done differently, TrackDefinitions can you make things align as much as possible.